src/server.c

```diff
@@ -6845,6 +6851,12 @@ sds genRedisInfoString(const char *section) {
         "# Cluster\r\n"
         "cluster_enabled:%d\r\n",
         server.cluster_enabled);
+
+        if (server.cluster_enabled) {
+            info = sdscatprintf(info,
+                    "total_cluster_links_buffer_limit_exceeded:%llu\r\n",
+                    server.cluster->stat_cluster_links_buffer_limit_exceeded);
+        }
```

Comment:
Let's move this into the `cluster info` command.

Reply:
Done